Automattic/newspack-plugin
# Update icons for web preview and popup action card

### All Submissions:






### Changes proposed in this Pull Request:

* Updates the icons for `WebPreview` and `PopupActionCard`
* Also updates npm packages to use the latest version of the Gutenberg icons

_Note: Category filtering icon is temporary, hopefully G2 will ship a more accurate icon in the future_

### How to test the changes in this Pull Request:

1. Check the web preview, notice the new icons
2. Check the campaigns, notice the new icons

### Other information:
